config/default.yml

```diff
@@ -1717,6 +1717,10 @@ Rails/ReadWriteAttribute:
   Include:
     - app/models/**/*.rb
 
+Rails/RefuteMethods:
+  Include:
+    - '**/*_test.rb'
```

Comment:
note that, this pattern was changed in rails/rails to `'**/test/**/*'` see https://github.com/rails/rails/pull/32648

Reply:
Thanks for the suggestion. I was wondering whether it was appropriate as a path pattern (`'**/test/**/*'`) for Rails applications because the configuration differs from Rails framework repository layout that has a test subdirectory for each components. There is only a little worry about false positives due to misidentification of test subdirectory path, so this path pattern seems that it can be applied to the Rails applications without problem.